src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html

```diff
@@ -32,6 +32,6 @@ <h3 class="edit-page-toolbar__page-title">
     <div class="ui-toolbar-group-right">
         <!-- Only show the button in edit mode -->
         <button pButton flat class="edit-page-toolbar__save" *ngIf="mode === 0" [disabled]="!canSave" [label]="dotMessageService.get('editpage.toolbar.primary.action')" (click)="save.emit($event)"></button>
-        <dot-edit-page-workflows-actions [inode]="pageState.page.workingInode" [label]="dotMessageService.get('editpage.toolbar.primary.workflow.actions')"></dot-edit-page-workflows-actions>
+        <dot-edit-page-workflows-actions [page]="pageState.page" [label]="dotMessageService.get('editpage.toolbar.primary.workflow.actions')"></dot-edit-page-workflows-actions>
```

Comment:
Are we 100% this is the best approach for this? because we're sending a lot of data to a component that literally only needs an string.

Reply:
it's needed because if we only send `workingInode` attr, then when a lock/unlock event happens then the `workingInode` attr won't be refreshed and the request won't be done to get the workflow actions. So by sending the `page` attribute we make sure to capture those changes

Reply:
Yup... I know... but, is that the best approach?